yeat/tests/data/configs/single.cfg

```diff
@@ -33,6 +33,22 @@
                 "Shigella_sonnei_53G"
             ],
             "mode": "single"
+        },
+        "penguin-default": {
+            "algorithm": "penguin",
+            "extra_args": "",
+            "samples": [
+                "Shigella_sonnei_53G"
+            ],
+            "mode": "single"
+        },
+        "velvet-default": {
+            "algorithm": "unicycler",
+            "extra_args": "",
```

Comment:
Typo: shouldn't this be `"algorithm": "velvet"`?

Reply:
Yes, good catch!